### PR TITLE
Standardized Readme

### PR DESCRIPTION
**Pull request by RichardLitt**
See https://github.com/ipfs/community/issues/124

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,29 @@
-![](https://ipfs.io/ipfs/QmbD39Nn6wUkf7HJucDMgZcSG1qd7RkvJrKM8a7iicR7Xq/ipfs-archives.png)
-
 # IPFS Archives
 
+![banner](https://ipfs.io/ipfs/QmbD39Nn6wUkf7HJucDMgZcSG1qd7RkvJrKM8a7iicR7Xq/ipfs-archives.png)
+
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
+[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
```
Comment (davidar) [inline README.md]: same comment as ipfs/apps#28 ;)

File: README.md [hunk 2]
```diff
@@ -1,10 +1,29 @@
-![](https://ipfs.io/ipfs/QmbD39Nn6wUkf7HJucDMgZcSG1qd7RkvJrKM8a7iicR7Xq/ipfs-archives.png)
-
 # IPFS Archives
 
+![banner](https://ipfs.io/ipfs/QmbD39Nn6wUkf7HJucDMgZcSG1qd7RkvJrKM8a7iicR7Xq/ipfs-archives.png)
```
Comment (davidar) [inline README.md]: ooh, very nice :+1:

File: README.md (hunk 1 above)
Comment (RichardLitt) [inline README.md]: Haha. Let's talk about it there.

**Comment by JesseWeinstein** [comment]
LGTM; why did you decide this needed a separate LICENSE file, but ipfs/apps didn't?

**Comment by RichardLitt** [comment]
I didn't add one to ipfs/apps because it was one of the first PRs I did, and I failed to add one to my own oversight. 

Changed License to point out that this is not a code repo. Should be good to go; one more pass after that last commit, anyone?

**Comment by JesseWeinstein** [comment]
Still LGTM.

**Pull request merged**